data/mods/BombasticPerks/perkdata/quick_recovery.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_PERK_QUICK_RECOVERY_INIT",
+    "eoc_type": "EVENT",
+    "required_event": "character_gains_effect",
+    "condition": {
+      "and": [
+        { "u_has_trait": "perk_quick_recovery" },
+        { "x_in_y_chance": { "x": 3, "y": 4 } },
+        { "or": [ { "compare_string": [ "downed", { "context_val": "effect" } ] } ] }
```

Comment:
I think this "or" isn't necessary.

Reply:
Originally it also did "or dazed" but I decided that was better than "kinda okay" so I removed it but not the or. I'll take that out.